tools/stress/src/org/apache/cassandra/stress/settings/CliOption.java

```diff
@@ -39,7 +39,8 @@
     PORT("The port to connect to cassandra nodes on", SettingsPort.helpPrinter()),
     SENDTO("-send-to", "Specify a stress server to send this command to", SettingsMisc.sendToDaemonHelpPrinter()),
     GRAPH("-graph", "Graph recorded metrics", SettingsGraph.helpPrinter()),
-    TOKENRANGE("Token range settings", SettingsTokenRange.helpPrinter())
+    TOKENRANGE("Token range settings", SettingsTokenRange.helpPrinter()),
+    CLOUDCONF("Configuration file of serverless Scylla cluster to connect to", SettingCloudConfig.helpPrinter())
```

Comment:
```suggestion
    CLOUDCONF("Configuration file of a serverless Scylla cluster to connect to", SettingCloudConfig.helpPrinter())
```

I also think that we should decide on official naming of serverless (is it "Scylla Serverless", "serverless Scylla", "Scylla Cloud serverless"?)

Reply:
@tzach @annastuchlik What is the correct name of serverless? How should this description of this flag look like?